code/index.html

```diff
@@ -25,16 +25,23 @@ <h1>Does the person have</h1>
       <select id="questions">
         <optgroup label="hair color">
```

Comment:
In the <head>, in title change to _Guess who_ so it look good on your browser tab.